translations/hu.json

```diff
@@ -0,0 +1,2984 @@
+          "Próbálj meg egy másik role-t."
+        ]
+      },
+      "3": {
+        "note": "Felemelve, amikor egy olyan guild-et választasz, ami már a szerverhez van kötve.",
+        "message": [
+          "Ez a guild már a szerverhez van kötve."
+        ]
+      },
+      "4": {
+        "note": "Felemelve, amikor egy olyan guild-et módosítasz, ami nincs benne az adatbázisban.",
+        "message": [
+          "Ez a guild nincs a szerverhez kötve."
+        ]
+      },
+      "5": {
+        "note": "Felemelve, amikor már egy eleve létező guild rangot adsz hozzá a szerverhez.",
+        "message": [
+          "Ez a rang már egy role-hoz van kötve."
+        ]
+      },
+      "6": {
+        "note": "Felemelve, amikor egy olyan role-t módosítasz, ami nincs benne az adatbázisban.",
+        "message": [
+          "Ez a rang nincs hozzáadva a szerverhez."
+        ]
+      },
+      "7": {
+        "note": "Felemelve, amikor egy már létező követelményt akarsz a guild-hoz adni.",
+        "message": [
+          "Ez a követelmény már a guild-hez van adva."
+        ]
+      },
+      "8": {
+        "note": "Felemelve, amikor egy olyan követelményt módosítasz, ami nincs az adatbázisban.",
+        "message": [
+          "Ez a követelmény nincs hozzáadva a guild-hoz."
+        ]
+      },
+      "9": {
+        "note": "Felemelve, amikor egy már létező követelményt akarsz a ranghoz adni.",
+        "message": [
+          "Ez a követelmény már a ranghoz van adva."
+        ]
+      },
+      "10": {
+        "note": "Felemelve, amikor egy olyan követelményt módosítasz, ami nincs az adatbázisban.",
+        "message": [
+          "Ez a követelmény nincs hozzáadva a ranghoz."
+        ]
+      },
+      "11": {
+        "note": "Felemelve, amikor egy követelmény maximum értékén túlhaladsz.",
+        "message": [
+          "A {maximum:,} maximumértéken a {requirement} követelménynél túlhaladtál."
+        ]
+      },
+      "12": {
+        "note": "Felemelve, amikor egy olyan guild-en próbálsz prémiumot aktiválni, ahol már az aktív.",
+        "message": [
+          "Ezen a szerveren már van prémium."
+        ]
+      },
+      "13": {
+        "note": "Felemelve, amikor egy olyan szerveren próbálsz prémiumot aktiválni, ahol egy guild-nek már van prémiuma egy felhasználótól.",
+        "message": [
+          "Ezen a szerveren van aktív prémium egy Skykings Premium tag által.\n",
+          "Prémium kódok nem aktiválhatóak ezen a szerveren."
+        ]
+      },
+      "14": {
+        "note": "Felemelve, amikor egy nem létező kódot akarsz aktiválni.",
+        "message": [
+          "Ez a prémium kód nem érvényes."
+        ]
+      },
+      "15": {
+        "note": "Felemelve, amikor egy eseményt akarsz elindítani, de ezen a szerveren már van elindítva esemény.",
+        "message": [
+          "Már fut egy esemény ezen a szerveren.\n",
+          "Ha már véget ért, kitörölheted a `/event delete` paranccsal."
```

Comment:
The whole command name should be translated, including top level
Should also be in backticks for `this effect`